docs/rules/camelcase.md

```diff
@@ -4,7 +4,7 @@ When it comes to naming variables, style guides generally fall into one of two c
 
 ## Rule Details
 
-This rule looks for any underscores (`_`) located within the source code. It ignores leading and trailing underscores and only checks those in the middle of a variable name. If ESLint decides that the variable is a constant (all uppercase), then no warning will be thrown. Otherwise, a warning will be thrown. This rule only flags definitions and assignments but not function calls.
+This rule looks for any underscores (`_`) located within the source code. It ignores leading and trailing underscores and only checks those in the middle of a variable name. If ESLint decides that the variable is a constant (all uppercase), then no warning will be thrown. Otherwise, a warning will be thrown. This rule only flags definitions and assignments but not function calls. In case of es6 `import` statements, this rule only targets the name of the variable that will be imported into the local module scope.
```

Comment:
This looks great to me! Could we make it `ES6` instead of `es6` for consistency with our documentation?